src/pages/transactiondetail/index.tsx

```diff
@@ -0,0 +1,558 @@
+import Container from "@components/commons/Container";
+import GradientCardContainer from "@components/commons/GradientCardContainer";
+import { FiCopy } from "react-icons/fi";
+import { AiOutlineDown, AiOutlineUp } from "react-icons/ai";
+import { NumericFormat } from "react-number-format";
+import { SearchBar } from "layouts/components/searchbar/SearchBar";
+import transactionDetailData from "mockdata/TransactionDetailData";
+import LinkText from "@components/commons/LinkText";
+import { truncateTextFromMiddle } from "shared/textHelper";
+import { Dispatch, SetStateAction, useState } from "react";
+import BigNumber from "bignumber.js";
+import { useUnitSuffix } from "hooks/useUnitSuffix";
+import { InfoIcon } from "@components/icons/InfoIcon";
+import Tooltip from "@components/commons/Tooltip";
+import clsx from "clsx";
+import { ConfirmCheck } from "@components/icons/ConfirmCheck";
+import { RejectedCross } from "@components/icons/RejectedCross";
+import { GreenTickIcon } from "@components/icons/GreenTickIcon";
+
+const data = transactionDetailData.transactionDetailData;
+
+function TransactionDetail() {
```

Comment:
Let's rename to `Transaction`, and rename file to use dynamic url, refer: https://nextjs.org/docs/routing/dynamic-routes

Reply:
renamed file to [tid].tsx and moved file to transactions folder